src/components/stepforms/widgets/MultiInputText.vue

```diff
@@ -265,6 +270,12 @@ export default class MultiInputTextWidget extends Vue {
   }
 }
 
+.widget-multiinputtext__multiselect--big.multiselect--active {
+  .multiselect__tags-wrap {
+    overflow-y: auto;
+  }
+}
```

Comment:
It's not clear for my why we should not use an overflow when we have less than 6 values.
Could you please add a comment to describe why this is necessary?

Reply:
Yes of course, it's because of tooltips they are not well displayed otherwise. I will let a comment